src/libraries/System.Security.Cryptography.Primitives/Directory.Build.props

```diff
@@ -3,6 +3,5 @@
   <PropertyGroup>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
`<IncludePlatformAttributes>` was added with `<UnsupportedOSPlatforms>` so perhaps it should be removed as well.

Reply:
Addressed in recent commits.  Thanks Maxim